src/screens/TermsConditionsScreen.tsx

```diff
@@ -110,12 +118,15 @@ export class TermsConditionsScreen extends React.PureComponent<Props> {
     );
   };
 
+  get canAgree() {
```

Comment:
I'm not sure what getter is doing here. Besides, I think it's simply an error to use getter in such a way.

Reply:
what is bad about getter?

Reply:
my bad, after reading about it I've changed my mind. I thought that what you did is some kind of a hack allowed in JS but it's actually a pretty common pracitce.